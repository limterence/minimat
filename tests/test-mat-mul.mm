/* test-op-matmul: matrix mult checks operands are conformable */
int main() {
  matrix u;
  u = [1.0, 2.0, 3.0, 4.0; 
      11.0, 12.0, 13.0, 14.0;];
  printmat(u * u);    /* matrix sizes do not conform for multiplication */
}
