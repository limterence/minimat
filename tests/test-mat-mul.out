illegal matrix dimensions for multiplication.  Exiting...
